tdesign/src/Link/MuiLink.tsx

```diff
@@ -47,6 +47,8 @@ export const NextLinkComposed = forwardRef<
       passHref={passHref}
       locale={locale}
     >
+      {/* SAFETY: `children` are spread in `other` */}
+      {/* eslint-disable-next-line jsx-a11y/anchor-has-content */}
       <a ref={ref} {...(other as any)} />
```

Comment:
FYI, this same code is the source of the mess in the marketing repo (copied from here). This needs to be updated to use new MUI recommended best practices. Also note the `any` came here due to profiling revealing it to significantly increase typechecking speed in files that imported `MuiLink`

Reply:
I remember the part about the `any` annotation to opt-out of typechecking. What is the other "mess" you're referring to? This component as a whole :smile: ?